tarantool/connection.py

```diff
@@ -1751,3 +1765,765 @@ def _check_features(self):
         features_list = [val for val in CONNECTOR_FEATURES if val in server_features]
         for val in features_list:
             self._features[val] = True
+
+    def crud_insert(self, space_name: str, values: Union[tuple, list], opts={}) -> CrudResult:
+        """
+        Inserts row through the `tarantool-crud <https://github.com/tarantool/crud#insert>`__ module.
+
+        :param space_name: The name of the target space.
+        :type space_name: :obj:`str`
+
+        :param values: The data for the crud module.
+        :type values: :obj:`tuple` or :obj:`list`
+
+        :param opts: The opts for the crud module.
+        :type opts: :obj:`dict`, optional
+
+        :rtype: :class:`~tarantool.crud.CrudResult`
+
+        :raise: :exc:`~tarantool.error.CrudModuleError`,
+            :exc:`~tarantool.error.DatabaseError`
+        """
+
+        assert isinstance(space_name, str)
+        assert isinstance(values, (tuple, list))
+        assert isinstance(opts, dict)
+
+        crud_resp = call_crud(self, "crud.insert", space_name, values, opts)
+
+        if crud_resp[1] is not None:
```

Comment:
Why re-raising hasn't been put to a `call_crud` method? You may use a couple of them if you need to (like `call_` and `call_many_`

Reply:
I thought about it, but there are methods in the Lua implementation that give not 2 values, but one. I noted this behavior with comments in the code `<In absence of an error, crud does not give variable err as nil (as in most cases)>`. Also, in some methods, not a table is returned, but a scalar value (for example, boolean, like `truncate`)

AFAIU, it is difficult to implement it just through `call_` and `call_many_`